src/nupic/algorithms/SpatialPooler.hpp

```diff
@@ -32,12 +32,14 @@
 #include <string>
 #include <vector>
 #include <capnp/message.h>
+#include <nupic/base/Serializable.hpp>
 #include <nupic/math/SparseBinaryMatrix.hpp>
 #include <nupic/math/SparseMatrix.hpp>
 #include <nupic/proto/SpatialPoolerProto.capnp.h>
 #include <nupic/types/Types.hpp>
 
 using namespace std;
+using namespace nupic::base::serializable;
```

Comment:
It's generally bad practice to have `using` statements in headers since it brings the contents into scope for anything that includes this header.